### PR TITLE
Add Builder::makeGlobal for nicer global creation

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

File: src/wasm-builder.h [hunk 1]
```diff
@@ -445,6 +445,15 @@ class Builder {
     return makeConst(value);
   }
 
+  // Module-level helpers
+  static Global* makeGlobal(Name name, WasmType type, Expression* init, bool mutable_) {
```
Comment (dschuff) [inline src/wasm-builder.h]: you could make the last argument an enum, and then you wouldn't need the comments everywhere.

Comment (kripken) [reply src/wasm-builder.h]: Good idea, my instincts here were wrong.

`NotMutable` or `Immutable`?

Comment (dschuff) [reply src/wasm-builder.h]: I'd say `Immutable`

Comment (kripken) [reply src/wasm-builder.h]: Ok, also a search shows that's more common, updated.

**Pull request merged**